ansible/roles/openshift_setup/tasks/main.yml

```diff
@@ -287,8 +287,10 @@
       msg: "Looking at oc cluster up command:  '{{ oc_cluster_up_cmd }}'"
 
   - name: Run oc cluster up to start the cluster
-    shell: "{{ oc_cluster_up_cmd }}"
+    shell: "{{ oc_cmd }} cluster down && {{ oc_cluster_up_cmd }}"
```

Comment:
How about doing oc cluster up and capturing if it succeeded?  Then have a ```when: oc_cluster_up is failed``` for the retries task.